jujugui/static/gui/src/app/components/user-profile/entity-list/test-entity-list.js

```diff
@@ -80,12 +80,15 @@ describe('UserProfileEntityList', () => {
 
   it('renders a list of charms', () => {
     var changeState = sinon.stub();
+    var getKpiMetrics = sinon.stub();
```

Comment:
Registering my complete distaste for mixing `var` and `let/const` in the same file, but doing it all the same.